server/views/pages/recommendations/ppcs/editCustodyGroup.njk

```diff
@@ -20,10 +20,12 @@
         <div class="govuk-grid-column-two-thirds-from-desktop">
             {% include '../../../partials/error-summary.njk' %}
 
-            <h1 class='govuk-heading-l'>{{ pageHeadings[page.id] }}</h1>
+            <h1 class='govuk-heading-l govuk-!-margin-bottom-7'>{{ pageHeadings[page.id] }}</h1>
 
-            <div class="govuk-text-list">
-                <h2 class='govuk-heading-m'>From Part A</h2>
+            {# We make the bottom margin smaller, as the default is too much when the text right after is as short as it is here #}
+            <h2 class='govuk-heading-m govuk-!-margin-bottom-2'>From Part A</h2>
+            {# Here we increase the margin, as otherwise this short paragraph seems too close to the radio button legend #}
+            <div class="govuk-text-list govuk-!-margin-bottom-7">
```

Comment:
This seems... odd, given that we are based on govuk styles, why do we need to deviate so far from the default styling. Unfamiliar with the `govuk-text-list` class but what is it doing that requires such extreme customisation over something like a `govuk-body`?